### ISSUE TITLE
FR: firewalld integration

### ISSUE THREAD

**Issue by firefoxlover** [localsend/localsend#1374]
### Is your feature request related to a problem? Please describe.

The feature to use a static port is really useful.

But currently when using Firewalld on Fedora Linux for example, there is no "localsend" profile to choose.

### Describe the solution you'd like

figuring out what kind of traffic needs to be allowed. Create a PR for a profile upstream in firewalld for that connection.

Is the port always 53317? Or just for me?

### Describe alternatives you've considered

manually enable the port. TCP and UDP, then it works.

**Comment by Tienisto** [comment]
Yes, 53317 is the current default port but I am thinking to use another secondary port if 53317 is already used.
See https://github.com/localsend/protocol/issues/12
